tests/commands/install_uninstall_test.py

```diff
@@ -759,6 +759,45 @@ def test_post_commit_integration(tempdir_factory, store):
         assert os.path.exists('post-commit.tmp')
 
 
+def test_post_merge_integration(tempdir_factory, store):
+    path = git_dir(tempdir_factory)
+    config = [
+        {
+            'repo': 'local',
+            'hooks': [{
+                'id': 'post-merge',
+                'name': 'Post merge',
+                'entry': 'touch post-merge.tmp',
+                'language': 'system',
+                'always_run': True,
+                'verbose': True,
+                'stages': ['post-merge'],
+            }],
+        },
+    ]
+    write_config(path, config)
+    with cwd(path):
+        #  create a simple diamond of commits for a non-trivial merge
+        open('init', 'a').close()
+        cmd_output('git', 'add', '.')
+        git_commit()
+
+        open('master', 'a').close()
+        cmd_output('git', 'add', '.')
+        git_commit()
+
+        cmd_output('git', 'checkout', '-b', 'branch', 'HEAD^')
+        open('branch', 'a').close()
+        cmd_output('git', 'add', '.')
+        git_commit()
+
+        cmd_output('git', 'switch', 'master')
```

Comment:
I know `git switch` is the nice new way of doing things, but there's realistically some packagers that use old git 2.x that don't have this yet -- could you use `git checkout -b` instead?

Reply:
Used git checkout, as I think this is what you meant.